lab_1_classify_by_unigrams/main.py

```diff
@@ -11,6 +11,10 @@ def tokenize(text: str) -> list[str] | None:
     :param text: a text
     :return: a list of lower-cased tokens without punctuation
     """
+    text = text.lower()
+    trans_table = str.maketrans({'.': None, ',': None, "'": None, ':': None, '?': None, '!': None, ' ': None, '1': None, '2': None, "3": None, "4": None, "5": None, "6": None, "7": None, "8": None, "9": None, "0": None})
```

Comment:
What if the text has more punctuation? Something like $#*&@"^
Try to find a more universal way to filter out everything but letters.